meson.build

```diff
@@ -1616,6 +1616,10 @@ if conf.get('ENABLE_RESOLVE') == 1
                          install_rpath : rootlibexecdir,
                          install : true)
         public_programs += [exe]
+
+        meson.add_install_script(meson_make_symlink,
+                         join_paths(bindir, 'systemd-resolve'),
+                         join_paths(rootsbindir, 'resolvconf'))
```

Comment:
That doesn't make much sense to cross from rootsbindir to bindir. `resolvconf` is in `/sbin` on Ubuntu/Debian, sp the second argument has to use rootsbindir. We can either say "who cares" and live with the fact that we'll have a dangling symlink until /usr is mounted, or move systemd-resolve to `/bin`. I don't care either way, maybe people who use split-usr can chime in?

@mbiebl, @martinpitt, does /sbin/resolvconf need to be usable before /usr is mounted?

Reply:
@keszybz on the systems that boot with systemd, our initramfs insures that /usr is mounted. So no. But there are things that use resolvconf state in the initramfs and hope that it will be transfered into the root filesystem, post initramfs. I have not yet checked, but it would be extra nice if there is e.g. path&service unit combos that process `/run/resolvconf/interface/$IFACE` files on boot. I didn't read all of the pull request, to check if this is done or not.

Reply:
no, we only provide command line compat with openresolv and debian's resolvconf, nothing else.

that said, if you run resolved in the initrd (which should be safe now) it should pass on its data 1:1 without isses to the main system